DevelopmentTools/HOMappingObject.h

```diff
@@ -44,9 +44,36 @@ struct HOTriggerTower
   int trg_fiber, trg_fiber_ch;
 };
 
+struct HOCalib
+{
+  int sector;
+  //FE
+  std::string rbx;
+  int rm, rm_fiber, fiber_ch;//rm is always 5, rm_fiber is always 1
+
+  //Calibration patch
+  int cpcol, cprow, cplc;
```

Comment:
rename variable and fix input:
  //patch panel
  int ppcol, pprow, pplc, dodec;
  std::string ppcpl;